odo/backends/sql.py

```diff
@@ -795,12 +807,19 @@ def append_table_to_csv(csv, selectable, dshape=None, bind=None, **kwargs):
                        merge(csv.dialect, kwargs))
     stmt = CopyToCSV(
         selectable,
-        os.path.abspath(csv.path),
+        os.path.abspath(csv.path) if csv.path is not None else None,
```

Comment:
I see where you have the `compiles` for `CopyToCSV` but not where path is used. What are the repercussions of the statement being executed without a path backing it?